skyportal/utils/offset.py

```diff
@@ -751,8 +757,24 @@ def get_nearby_offset_stars(
                            {radius_degrees}))
                 """
 
-    g = GaiaQuery()
-    r = g.query(query_string)
+    try:
```

Comment:
PR looks good.

I'd probably combine this into a single try loop, but not a big deal.

```python
result = None
for retry in range(2):
    try:
        g = GaiaQuery()
        result = g.query(query_string)
    except Exception as e:
        log(f'Gaia failed [{retry}] ... other info')

if result is None:
    return ...
```

Reply:
Good idea. Implemented.